scripts/serviced-storage.py

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env python
 
 # zenoss-inspector-info
-# zenoss-inspector-deps serviced-confg.sh
+# zenoss-inspector-deps etc-default-serviced.sh
 
 def main():
-    with open("serviced-config.sh.stdout", 'r') as f:
+    with open("etc-default-serviced.sh.stdout", 'r') as f:
```

Comment:
Might want to stick with serviced-config.sh.stdout.

I could be wrong, but I think using etc-default-serviced.sh.stdout will resort in a false alarm in the case that SERVICED_FS_TYPE is not defined /etc/default/serviced. In a case like, serviced-config.sh.stdout should have the default value that would be used by CC.